css/layout.css

```diff
@@ -0,0 +1,65 @@
+/* layout.css */
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: #f0f0f0;
+}
+
+header {
+  background-color: #d32f2f;
+  color: white;
+  padding: 1rem;
+  text-align: center;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 1rem;
+}
+
+nav button {
+  background-color: #f44336;
+  border: none;
+  color: white;
+  padding: 0.5rem 1rem;
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
+
+nav button:hover {
+  background-color: #e53935;
+}
```

Comment:
Most of these are _styling_ elements and should be in your `styles.css`. Things in `layout.css` would be classes that affect page layout such as:

```
.row {
  display: flex;
}

.column-full {
  flex-basis: 100%;
}
```